exercises/Robert_Applin/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,203 @@
+/**
+ * Skeleton main routine
+ */
+
+#include <algorithm>
+#include <cctype>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+using WordCount = std::vector<std::pair<std::string, std::size_t>>;
+
+/**
+ * @return An iterator pointing to the next letter contained
+ *         in the supplied set.
```

Comment:
Good that you described the `@return` of the function but what about the `@params`s? And a summary of what the function does.

Reply:
Also update other places where this is true.